### DISCUSSION TITLE
征集 ONES Group 的 logo 设计

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#19] [日常讨论]
如题，有好的设计可以提出来~

**Comment by 520077778** [comment]
![1](https://user-images.githubusercontent.com/61972485/158021952-9551c8eb-24d1-41cb-9c03-ae1410ca3590.jpg)
![2](https://user-images.githubusercontent.com/61972485/158021957-ce76a66c-5214-43ad-ae23-9de44cf51ef7.jpg)

**Comment by oooohwell** [comment]
![ONES logo-2](https://user-images.githubusercontent.com/101536912/159115386-9f03b2b3-4923-41fd-ba66-7bf9aebf1fd4.png)

**Reply by oooohwell** [reply]
![ONES Group logo-1](https://user-images.githubusercontent.com/101536912/159144485-df30f4b9-e0a5-44ba-a274-2ba3b1b4817c.png)
![ONES Group logo-3](https://user-images.githubusercontent.com/101536912/159144538-d161aa60-5368-4b72-ac5f-97c257b0fcd7.png)

我参考了其他品牌的logo又试了试别的样式，第一版图标和字母在一起感觉有些过于臃肿，可能这几种会好一些，我都传上来，老师们自己看看更喜欢哪个吧吧

**Reply by will-ww** [reply]
太棒了，鲜花！

**Comment by QuakeWang** [comment]
![ONES](https://user-images.githubusercontent.com/45645138/159154031-a3b8b8df-2313-4fa2-8aee-e5149c126530.png)
@will-ww 王老师，看一下这样设计如何~~

**Comment by will-ww** [comment]
感谢大家！

**Comment by AliceCodeZhang** [comment]
![image](https://user-images.githubusercontent.com/31360564/167252771-7d83285c-a477-4368-bfba-2ea1e444904b.png)



LOGO: 左边外看是一个圆，圆和英文的O类似，在中华民族文化里面通常象征着“圆满”，圆则满,满则圆，心有圆满便安宁不争,便以和为贵，左边留了一个口，表示这个圆是开放的。开放、共享、和谐、共进正是开源文化可贵之处。
在这个圆里面其实是一个矩阵，这个矩阵通常可以用函数ones（）来构造，其实就是全1矩阵，如果把任何一个参与开源的个体比喻成一个数字，这个圆里面的矩阵代表开源世界，任何数字和这个矩阵相乘之后，依旧保留的是个体本来的样子，因为任何数x1 都等于他自己，表示开源世界尊重每个参与者的个性特征。同时，矩阵可以做很多数学运算，各种运算代表这个体之间交互之后得出来的结果，个体可以为整个开源世界添砖加码，个体和个体之间也可以相互碰撞产出更多的可能，无限的可能来自开源的你我他。右边是运算的结果就是一个整体的ones group

**Reply by AliceCodeZhang** [reply]
刚才好像上传错了issue,重新贴一下：）

**Comment by wangleo61** [comment]
好酷呀

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Yanbin ***@***.***>
发送时间: Saturday, May 7, 2022 7:44 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: Re: [kaiyuanshe/ONES] 征集 ONES Group 的 logo 设计 (Discussion #19)


[image]<https://user-images.githubusercontent.com/31360564/167252771-7d83285c-a477-4368-bfba-2ea1e444904b.png>

LOGO 应该是左边，我光画了这个发现好像不好看，就加了ones 在右边

LOGO: 外看是一个圆，圆和英文的O类似，在中华民族文化里面通常象征着“圆满”，圆则满,满则圆，心有圆满便安宁不争,便以和为贵，左边留了一个口，表示这个圆是开放的。开放、共享、和谐、共进正是开源文化可贵之处。
在这个圆里面其实是一个矩阵，这个矩阵通常可以用函数ones（）来构造，其实就是全1矩阵，如果把任何一个参与开源的个体比如一个数字，这个圆里面的矩阵代表开源世界，任何数字和这个矩阵相乘之后，依旧保留的是个体本来的样子，因为任何数x1 都等于他自己。同时，矩阵可以做很多数学运算，各种运算代表这个体之间交互之后得出来的结果，个体可以为整个开源世界添砖加码，个体和个体之间也可以相互碰撞产出更多的可能，无限的可能来自开源的你我他

―

**Comment by will-ww** [comment]
由于 ONES 已经在多个领域做了商标注册，如果后面我们也需要注册的话，可能还需要同时做一些演绎，下面几个供参考~

![image](https://user-images.githubusercontent.com/15010826/168000974-9eb39e83-dc2a-4416-9a69-f530c761e9ee.png)

![image](https://user-images.githubusercontent.com/15010826/168000997-702669e0-7682-4890-b894-2264da99b79c.png)

**Reply by AliceCodeZhang** [reply]
赞 ‘开源善治’

**Comment by will-ww** [comment]
来自梁尧的信息：ones 在多个类别已被注册（尤其是41教育娱乐和42科技服务类），如果申请会因为“同他人在同一种商品或者类似商品上已经注册的或者初步审定的商标相同或者近似的”而被驳回，即使增加了“group”也会存在同样问题

![faf9699f73419a3ebc9957589a58c4f](https://user-images.githubusercontent.com/15010826/168002924-92c5d4e5-d9ad-4c29-a453-5b019d1838fc.png)

![65791040800380d5b719ed65bb788e6](https://user-images.githubusercontent.com/15010826/168003096-15dda4bd-e2e9-49b6-879f-7c4e663e1ce4.png)

![4eeee148442f0bf39748fa050007ebe](https://user-images.githubusercontent.com/15010826/168003125-ffb8a373-e6e9-4a57-9ed8-d9e3efee78b5.png)

类别对照：https://www.nisifenlei.com/

**Comment by 520077778** [comment]
![QQ截图20220606014451](https://user-images.githubusercontent.com/61972485/172063447-d76b86e0-8d29-49fa-95ca-3f43f6d8145b.jpg)
![QQ截图20220606014502](https://user-images.githubusercontent.com/61972485/172063453-bc2acb0f-0c8b-4f79-b111-c839acc9c293.jpg)
![QQ截图20220606014510](https://user-images.githubusercontent.com/61972485/172063460-6861f6eb-24e6-456d-a8bd-a43561a56276.jpg)

**Comment by wangleo61** [comment]
+1,马斯克说善良很重要
原来开源社搞过开源向善

发件人: Yanbin ***@***.***>
发送时间: Tuesday, May 17, 2022 2:24 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 征集 ONES Group 的 logo 设计 (Discussion #19)


赞 ‘开源善治’

―
